friendsofphp/php-cs-fixer/2.19/manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+    "aliases": ["cs-fixer", "php-cs-fixer"],
+    "copy-from-recipe": {
+        ".php-cs-fixer.dist.php": ".php-cs-fixer.dist.php"
+    },
+    "gitignore": [
+        "/.php-cs-fixer.php",
+        "/.php_cs.cache"
```

Comment:
Nor sure about that @keradus

Reply:
PHP CS Fixer uses:
- `.php_cs.cache` for 2.x
- `.php-cs-fixer.cache` for 3.x

Reply:
Ah so we need to keep `.php_cs.cache` for 2.x and add another recipe for 3.x with `.php-cs-fixer.cache` ?

Reply:
yes

Reply:
Ok done

Reply:
The name is configurable though. I'd suggest to configure PHP CS Fixer to use the new cache file name so we don't need different recipes for 2.19 and 3.0.

Reply:
@derrabus good point for maint, but wouldnt it be best to stick with the conventions used in `3.0`? thinking in terms of DX and consistency - e.g. `maker-bundle` will use `php-cs-fixer` `3.0` but does not depend on recipes. However in our test cases, we _do_ depend on recipes and I can see the potential of the bundle vs tests being out of sync. 

Another use case would be for other bundles e.g. `symfonycasts` || `knplabs` that try to stick as close as possible Symfony conventions. It would be nice to keep `php-cs-fixer`'s configuration && `.gitignore` inline with Symfony w/o having to use custom naming conventions.

Reply:
Maybe sth like this: https://github.com/symfony/recipes/pull/942/files#r625770976

then, 2.x and 3.x will use same name of cache file
(and that line can be removed on 3.x, as `.php-cs-fixer.cache` is default name on 3.x)

Reply:
@jrushlow I don't quite get why MakerBundle would have an opinion on how PHP CS Fixer's cache file is named.

What @keradus suggested is exactly what I meant. The 2.19 release prepares the application for the transition to 3.0. If we can configure CS Fixer 2.19 to behave like 3.0, we should do that.